naas/requests/project_subscribers.py

```diff
@@ -0,0 +1,64 @@
+import json
+from naas.client import Client
+from naas.errors import RecordNotFoundError
+
+
+class ProjectSubscribers:
+
+    @staticmethod
+    def list_by_project_id(project_id, params=None):
+        """
+        Retrieve the list of subscribers by project
+
+        :param params: dict
+        :return: Response
+        """
+        if params is None:
+            params = {}
+
+        rel = Client.rel_for('rels/project-subscribers')
+        route = Client.routes().route_for(rel)
+        url = route.url_for(args={**params, **{'project_id': project_id}})
+        request = Client.get(url)
+        return request
+
+    @staticmethod
+    def retrieve_by_project_id(project_id, params=None):
```

Comment:
@arthuroe - this needs to have the `id` of the subscriber in the arguments so it can retrieve the instance.